packages/grafana-ui/src/components/GraphNG/GraphNG.tsx

```diff
@@ -65,7 +65,7 @@ export const GraphNG: React.FC<GraphNGProps> = ({
   const hasLegend = useRef(legend && legend.displayMode !== LegendDisplayMode.Hidden);
   const alignedFrame = alignedFrameWithGapTest.frame;
   const compareFrames = useCallback(
-    (a: DataFrame, b: DataFrame) => compareDataFrameStructures(a, b, ['min', 'max']),
+    (a: DataFrame, b: DataFrame) => compareDataFrameStructures(a, b, []), //  ['min', 'max']),
```

Comment:
ignoring min/max should not be necessary since they should not update with every query